dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -34,6 +34,26 @@ const adStylesDynamic = emoCss`
 
 	}
 
+	.ad-slot--im {
```

Comment:
These styles now live with the other dynamic ad styles in `ElementContainer` (at least for now...)